arborist/server.go

```diff
@@ -534,6 +534,7 @@ func (server *Server) handleAuthRequest(w http.ResponseWriter, r *http.Request,
 				return
 			}
 			if !rv.Auth {
+				server.logger.Debug("anonymous user is unauthorized")
```

Comment:
Just wondering, is there any specific use case, that is driving us to add this log? Can we add it in the PR description?

Reply:
no. just making debugging easier